sources/playground/default_client.c

```diff
@@ -6,16 +6,7 @@
  */
 
 #include "cqlrt.h"
-#include "go.h"
-
-// super cheesy error handling
-#define _E(c, x) if (!(c)) { \
-  printf("!" #x "%s:%d\n", __FILE__, __LINE__); \
-  goto error; \
-}
-
-#define E(x) _E(x, x)
-#define SQL_E(x) _E(SQLITE_OK == (x), x)
+#include EXAMPLE_HEADER_NAME
```

Comment:
This could really use a comment about how it gets resolved.

Reply:
I tried to use `c_include_namespace` and/or `--c_include_path` but then it broke the cc call.

Renaming the macro to something more self explanatory `#include HEADER_FILE_FOR_SPECIFIC_EXAMPLE`.

Reply:
The question is not what it is but how it will get populated.  It can't be a #define right because macros are not expanded in that context.  The thing to document is that the play script is going to replace it (somehow).